JWBestPracticeApps/Test/Test/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
Seems some extra files made their way to the PR that were used for testing purposes, these should not be included.